### PR TITLE
Flutter for Android/iOS/Xamarin page tweaks

### PR DESCRIPTION
**Pull request by chalin**
- Mainly page markdown cleanup (switched to using markdown numbered-list syntax and code fences). There is no change in appearance for the affected pages.
- Almost no content change except for one small adjustment in web-analogs (I'll point it out using an inlined comment).
- Actually, I ended up having to convert two files from DOS line endings to unix.

cc @Sfshaza

### CODE REVIEW

**Comment by chalin** [comment]
@Sfshaza - Test are passing. The build is now failing but only because of

> External link https://adwords.google.com/home/resources/adwords-app.html failed: 404 No error

File: web-analogs.md [hunk 1]
```diff
@@ -16,25 +16,25 @@ arranging components of an application's UI. It maps HTML/CSS code snippets to
 their Flutter/Dart code equivalents.
 
 The examples assume:
-* The HTML document starts with a modern HTML DOCTYPE, and the CSS box model
-for all HTML elements is set to
-[`border-box`](https://css-tricks.com/box-sizing/),
-for consistency with the Flutter model.
-   {% prettify css %}<!DOCTYPE html>
```
[the remark this answers is not in the document]
Comment (Sfshaza) [reply web-analogs.md]: yes

**Comment by Sfshaza** [comment]
LGTM

**Comment by chalin** [comment]
> I agree about the challenge of external links. Flutter has always done it that way, but I welcome turning those from errors to warnings.

I've opened https://github.com/flutter/flutter/issues/19752 to track that.

**Comment by chalin** [comment]
Finally, Travis is green!

**Comment by Sfshaza** [comment]
LGTM

**Pull request merged**